setup.cfg

```diff
@@ -1,7 +1,15 @@
 # flake8 does not support pyproject.toml, see:
 # https://github.com/PyCQA/flake8/issues/234
 [flake8]
-exclude = docs
+exclude = docs,venv
 max-line-length = 100
 
+[mypy]
+files = src,test
+ignore_missing_imports = true
+exclude = docs,build
 
+[tool:pytest]
+addopts = --cov=src/mypackage --cov-report html --cov-report term-missing -v
+testpaths =
+    test
```

Comment:
The reason for having this inside `pyproject.toml` is that the long term goal should be to remove `setup.cfg`. See: https://github.com/pypa/setuptools/issues/3214

Reply:
I see. Since `flake8` still doesn't support `pyproject.toml` we need `setup.cfg` (or some other config format). I think it would be good to have everything in one file (i.e `pyproject.toml`), but if that cannot happen then I think it would be best to put tooling configurations (such as `mypy`, `flake8` and `pytest`) in one place. What do you think about that? Otherwise I think we should make a [`.flake8` config file](https://flake8.pycqa.org/en/latest/user/configuration.html#configuration-locations) to make it clear that the configurations are `flake8` only.

You can add pytets configurations in `pyproject.toml` see https://docs.pytest.org/en/6.2.x/customize.html#pyproject-toml

Reply:
I guess if flake8 is the only issue lets make a .flake8 config file

Reply:
https://github.com/jorgensd/reproducibility/pull/33/commits/8b6a585fa9efee26fc6d6e20c3b0e65b0c9f1f26